cmd/verrazzano-operator/main.go

```diff
@@ -152,13 +152,12 @@ func main() {
 	if startController {
 		// start the controller
 		if err = controller.Run(2); err != nil {
-			glog.Fatalf("Error running controller: %s", err.Error())
+			zap.S().Fatalf("Error running controller: %s", err.Error())
 		}
 	}
 
-	glog.Info("Waiting for api server to exit")
+	zap.S().Infow("Waiting for api server to exit")
```

Comment:
what is the w in Infow?

Reply:
There are two log fuctions we may use to replace glog.Info:
```
// Infow logs a message with some additional context. The variadic key-value
// pairs are treated as they are in With.
func (s *SugaredLogger) Infow(msg string, keysAndValues ...interface{}) 


// Info uses fmt.Sprint to construct and log a message.
func (s *SugaredLogger) Info(args ...interface{}) {
```
The Infow is taking a typed/string msg which I feel closer to the original one. the 'w' suffix means 'With' which allows optional fields.